### PR TITLE
Chore: related module bugfixes

### PR DESCRIPTION
**Pull request by anerolabs**
Product card/text scaling
https://trello.com/c/av8b8P1E/140-related-fix-product-card-scaling-text-scaling

Switch to thumbnail source images
https://trello.com/c/1PD6jhu7/143-related-switch-to-thumnail-source-urls-for-product-card-images

Post-cache implementation bug fixes
https://trello.com/c/NKWBBXNU/141-related-post-cache-refactor-bug-fixes

Compare modal fixes
https://trello.com/c/b9q9KhBl/108-related-chore-compare-modal-fixes

### CODE REVIEW

File: src/components/overview/Overview.jsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ const Overview = ({productId}) => {
 
   return (
     <>
-      <div id='announce'><b>&nbsp;FREE SHIPPING&nbsp;+&nbsp;RETURNS,&nbsp; SAVE UP TO 30% ON THE SELECTED ITEMS</b></div>
+      <div id='announce'><b>&nbsp;FREE SHIPPING&nbsp;+&nbsp;RETURNS,&nbsp; SAVE UP TO 30% ON SELECT ITEMS</b></div>
```
Comment (lbc1013) [inline src/components/overview/Overview.jsx:129]: thank you for changing this!

File: src/App.jsx [hunk 2]
```diff
@@ -128,7 +121,7 @@ class App extends React.Component {
 
   addItemToOutfit(productToAdd) {
     const outfitItems = this.state.outfitItems;
-
+    console.log('addy item', productToAdd);
```
Comment (lbc1013) [inline src/App.jsx]: console.log here!

File: src/components/related/YourOutfitList.jsx [hunk 3]
```diff
@@ -113,7 +113,7 @@ class YourOutfitList extends React.Component {
           <div className='product-card-list'>
           <div className='product-card add-outfit-card'
             onClick={() => { this.handleAddToOutfitClick(productId) }}>
-            <img className='add-outfit-icon' src='https://tinyurl.com/3hspumek'></img>
+            <img className='add-outfit-icon' src='https://gcdn.pbrd.co/images/pP7Ot4WmrM2q.png?o=1'></img>
```
Comment (lbc1013) [inline src/components/related/YourOutfitList.jsx:116]: 👍

**Review by lbc1013 — APPROVED** [review]
looks good to me👍

File: src/components/overview/Overview.jsx (hunk 1 above)
Comment (anerolabs) [reply src/components/overview/Overview.jsx:129]: Hope you didn't mind! I missed it when you PR'd it originally

**Pull request merged**